# HG changeset patch
# User Rik <rik@octave.org>
# Date 1390446520 28800
#      Wed Jan 22 19:08:40 2014 -0800
# Node ID 313ebc905e38d970a4f36767f88841762a073a7e
# Parent  15487cae60a3182f6edab171ae883c18b14dc12c
fill.m: Add titles to %!demos.

* fill.m: Add titles to %!demos.

diff --git a/scripts/plot/draw/fill.m b/scripts/plot/draw/fill.m
--- a/scripts/plot/draw/fill.m
+++ b/scripts/plot/draw/fill.m
@@ -166,21 +166,35 @@ endfunction
 %! clf;
 %! t1 = (1/16:1/8:1) * 2*pi;
 %! t2 = ((1/16:1/8:1) + 1/32) * 2*pi;
 %! x1 = sin (t1) - 0.8;
 %! y1 = cos (t1);
 %! x2 = sin (t2) + 0.8;
 %! y2 = cos (t2);
 %! h = fill (x1,y1,'r', x2,y2,'g');
+%! title ({'fill() function'; 'cdata specified with string'});
+
+%!demo
+%! clf;
+%! t1 = (1/16:1/8:1) * 2*pi;
+%! t2 = ((1/16:1/8:1) + 1/32) * 2*pi;
+%! x1 = sin (t1) - 0.8;
+%! y1 = cos (t1);
+%! x2 = sin (t2) + 0.8;
+%! y2 = cos (t2);
+%! h = fill (x1,y1,1, x2,y2,2);
+%! title ({'fill() function'; 'cdata = row vector produces FaceColor = "flat"'});
 
 %!demo
 %! clf;
 %! x = [0 0
 %!      1 0.5
 %!      1 0.5
 %!      0 0];
 %! y = [0 0
 %!      0 0
 %!      1 0.5
 %!      1 0.5];
 %! c = [1 2 3 4]';
 %! fill (x, y, c);
+%! title ({'fill() function'; 'cdata = column vector produces FaceColor = "interp"'});
+
